sympy/core/containers.py

```diff
@@ -49,7 +49,7 @@ class Tuple(Basic):
 
     def __new__(cls, *args, **kwargs):
         if kwargs.get('sympify', True):
-            args = ( sympify(arg) for arg in args )
+            args = (sympify(arg) for arg in args)
```

Comment:
Should this not use `strict=True`?

Reply:
After changing this, I saw a lot of breakdowns because some sympy code is abusing this.
So I’d deal with other PR about that.